driver/test_kubeflow_workloads.py

```diff
@@ -65,6 +65,12 @@ def pytest_filter(request):
     return f"-k '{filter}'" if filter else ""
 
 
+@pytest.fixture(scope="session")
+def include_gpu_tests(request):
+    """Retrieve the `--include-gpu-tests` flag from Pytest invocation."""
+    return True if request.config.getoption("--include-gpu-tests") else False
```

Comment:
based on the spec, we should include the following message when gpu flag is not set:
```
Note that only CPU tests will be run. In order to run tests that use an NVIDIA GPU, use the `--include-gpu-tests` flag e.g. `tox -e kubeflow-local -- --include-gpu-tests`. To learn more, use `--help`.
```

Reply:
Good catch!